packages/admin-manifest/package.json

```diff
@@ -0,0 +1,33 @@
+{
+	"name": "@wordpress/admin-manifest",
+	"version": "1.0.0",
+	"description": "Dynamically creates a Web App manifest and registers the service worker for the admin.",
```

Comment:
Can you mark it as private so it doesn't get published to npm until it's ready for prime time?

Reply:
Oh, sure, I wasn't aware of that.